packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/ExplorerDescriptionProvider.java

```diff
@@ -130,7 +130,7 @@ private String getExplorerTreeId(List<?> expandedObjects, List<?> activatedFilte
                 .map(id -> URLEncoder.encode(id, StandardCharsets.UTF_8))
                 .toList();
 
-        return "explorer://?expandedIds=[" + String.join(",", expandedObjectIds) + "]?&activeFilterIds=[" + String.join(",", activatedFilterIds) + "]";
+        return "explorer://?expandedIds=[" + String.join(",", expandedObjectIds) + "]&activeFilterIds=[" + String.join(",", activatedFilterIds) + "]";
```

Comment:
This fix shouldn't be part of this PR, it fixes an issue on master when selecting elements in the explorer.
I added a quick fix here to make sure this PR can be properly tested

Reply:
Ok